packages/fuel-gauge/src/doc-examples.test.ts

```diff
@@ -277,67 +277,14 @@ describe('Doc Examples', () => {
 
     const receiver = Wallet.generate({ provider });
 
-    const AbiInputs: JsonAbi = {
-      types: [
-        {
-          typeId: 0,
-          type: 'bool',
-          components: null,
-          typeParameters: null,
-        },
-        {
-          typeId: 1,
-          type: 'struct B512',
-          components: null,
-          typeParameters: null,
-        },
-        {
-          typeId: 2,
-          type: '[_; 3]',
-          components: [
-            {
-              name: '__array_element',
-              type: 1,
-              typeArguments: null,
-            },
-          ],
-
-          typeParameters: null,
-        },
-      ],
-      functions: [
-        {
-          inputs: [
-            {
-              name: 'data',
-              type: 2,
-              typeArguments: null,
-            },
-          ],
-          name: 'main',
-          output: {
-            name: '',
-            type: 0,
-            typeArguments: null,
-          },
-          attributes: null,
-        },
-      ],
-      loggedTypes: [],
-      configurables: [],
-      messagesTypes: [],
-    };
```

Comment:
Do you know the original rational behind having the ABI here in the first place?

(Not an issue, just intrigued)